codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -132,8 +150,10 @@ class FileCodesignVisitor {
           zipEntity: entity,
           entitlementParentPath: entitlementParentPath,
         );
+      } else if (childType == FileType.binary) {
+        await visitBinaryFile(binaryFile: entity as File, entitlementParentPath: entitlementParentPath);
       }
-      log.info('Child file of direcotry ${directory.basename} is ${entity.basename}\n');
+      log.info('Child file of direcotry ${directory.basename} is ${entity.basename}');
```

Comment:
```suggestion
      log.info('Child file of directory ${directory.basename} is ${entity.basename}');
```

Reply:
my man I really spelled a lot of things incorrectly. Thanks for catching them, it's impressive Christopher noticed all these details!